content/webapp/pages/page.js

```diff
@@ -105,7 +108,20 @@ export class Page extends Component<Props> {
         imageUrl={page.image && convertImageUri(page.image.contentUrl, 800)}
         imageAltText={page.image && page.image.alt}
       >
-        <ContentPage id={page.id} Header={Header} Body={<Body body={body} />} />
+        <ContentPage
+          id={page.id}
+          Header={Header}
+          Body={<Body body={body} pageId={page.id} />}
+        />
+
+        {/* Display closures on the library page */}
```

Comment:
Quite a lot of brittle bespoke code for things that future selves will probably not like.

Could this be split up into smaller task, potentially looking at these pieces individually and maybe coming up with a more scalable solution?

Reply:
We weren't totally in love with this particular shim, but seemed like the least esoteric of the options available:

1. Singleton type slice
2. HTML serialiser
3. hardcode in the template

Have we overlooked another option?

Reply:
Could you have it as a `collectionVenue` slice, and then add a label of `closing` or something?